### DISCUSSION TITLE
Suggestion: combine lists with Polkadot

### DISCUSSION THREAD

**Discussion by sime** [MetaMask/eth-phishing-detect#11725] [Ideas]
I don't have a concept how this can be done technically, though Polkadot often is aware about phishing campaigns before they arrive to the MetaMask blocklist.

https://github.com/polkadot-js/phishing

**Comment by dubstard** [comment]
tl.dr - they pay their staff for the submissions, so quality is great.
they mostly are interested in DOT/SAMA ecosystem, but as there ate often multi wallet stealers a lot of the data is suitable for MMs blacklist as well.


----------------------

https://medium.com/polkadot-network/polkadots-anti-scam-initiatives-1660ef5787b7
You can read more about it here.

**Comment by tayvano** [comment]
crosslinking this to recent discussion in [slack](https://consensys.slack.com/archives/C03KW2BGB8T/p1677139380266489)

> in my opinion we should have a cron job that grabs all their new submissions, checks for dupes and auto merges each 24 hours or so, essentially leveraging the DOT SAMA anti scam imitative s potential to the fullest extent, as the user base of MM is siginificantly larger than DOTs, so much more users will get protexction much faster.
